blocks/link-group/link-group.css

```diff
@@ -0,0 +1,92 @@
+.link-group-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.link-group-wrapper {
+  width: 100%;
+  border-bottom: 0.5px #a9b4be solid;
+}
+
+.link-group-container > div {
+  margin: 0;
+}
+
+.block.link-group ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: none;
+}
+
+.block.link-group ul li {
+  padding: 8.5px;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.block.link-group.active ul {
+  display: block;
+}
+
+.block.link-group strong {
```

Comment:
The font definition should be using these design tokens, i.e. var(--...) :
  --udexTypographyNavigationMediumXLFontSize: 16px;
  --udexTypographyNavigationMediumXLFontFamily: "72 Brand", "Arial", "Helvetica", "sans-serif";
  --udexTypographyNavigationMediumXLFontWeight: 500;
  --udexTypographyNavigationMediumXLLineHeight: 18px;